src/services/lease-service/routes/rentalObjects.ts

```diff
@@ -0,0 +1,202 @@
+import KoaRouter from '@koa/router'
```

Comment:
IMO name this file rental-objects.ts to continue with the style of kebab-case

Reply:
<img width="334" alt="Screenshot 2025-07-01 at 11 47 06" src="https://github.com/user-attachments/assets/88eecb91-80d8-44bd-8c24-aa99a6432ec3" />